gems/jwt/2.5/jwt.rbs

```diff
@@ -0,0 +1,48 @@
+module JWT
+  interface _Algo
+    def alg: () -> ::String
+    def sign: (data: ::String, signing_key: untyped) -> ::String
+    def valid_alg?: (String alg_to_validate) -> bool
+    def verify: (data: untyped, signature: ::String, verification_key: untyped) -> bool
+  end
+
+  def self?.encode: (untyped payload, untyped key, ::String | _Algo algorithm, ?::Hash[::String | ::Symbol, untyped] header_fields) -> ::String
+
+  def self?.decode: (String jwt, ?untyped? key, ?bool verify, ?::Hash[Symbol, untyped] options) ?{ (Hash[::String, untyped] header, ?untyped payload) -> untyped } -> [untyped, Hash[::String, untyped]]
```

Comment:
```suggestion
  def self?.decode: (String jwt, ?untyped? key, ?boolish verify, ?::Hash[Symbol, untyped] options) ?{ (Hash[::String, untyped] header, ?untyped payload) -> untyped } -> [untyped, Hash[::String, untyped]]
```

How about using `boolish` instead of `bool`?
https://github.com/ruby/rbs/blob/30e800622a2d7217f822beb2693bdff16f528193/docs/syntax.md#bool-or-boolish

`decode` method receives anything as `verify`, and the argument is used only for `if`/`unless` conditions. So I think we have no reason to strict it to `true` or `false`.